components/script/dom/identityhub.rs

```diff
@@ -86,12 +90,16 @@ impl Identities {
             Backend::Dx12 => &mut self.dx12_hub,
             #[cfg(any(target_os = "ios", target_os = "macos"))]
             Backend::Metal => &mut self.metal_hub,
+            #[cfg(any(target_os = "linux", target_os = "windows"))]
+            Backend::Gl => &mut self.gl_hub,
             _ => &mut self.dummy_hub,
         }
     }
 
     fn hubs(&mut self) -> Vec<(&mut IdentityHub, Backend)> {
         vec![
+            #[cfg(any(target_os = "linux", target_os = "windows"))]
+            (&mut self.gl_hub, Backend::Gl),
             #[cfg(any(target_os = "linux", target_os = "windows"))]
             (&mut self.vk_hub, Backend::Vulkan),
             #[cfg(target_os = "windows")]
```

Comment:
I think we need to stick to ordering here to ...

Reply:
Sorry i missed this one